.github/workflows/python-package.yml

```diff
@@ -0,0 +1,59 @@
+name: Python package
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+env:
+  LINUX_BASEKIT_URL: https://registrationcenter-download.intel.com/akdlm/irc_nas/17769/l_BaseKit_p_2021.2.0.2883_offline.sh
```

Comment:
Why use this hard coded url? Instead we should just use the [APT oneAPI repo](https://software.intel.com/content/www/us/en/develop/documentation/installation-guide-for-intel-oneapi-toolkits-linux/top/installation/install-using-package-managers/apt.html?fireglass_rsn=true). Github actions allow using password-less `sudo`.

Another option can be to use nightly builds of dpcpp from https://github.com/intel/llvm/releases. I am working on a set of clean ups to our build system that will make it very easy to build with the open source dpcpp.